### DISCUSSION TITLE
New fail animation is dangerous

### DISCUSSION THREAD

**Discussion by RadekMojzis** [ppy/osu#15322] [Ideas]
First time i failed with the new laser fail animation (screen flashes bright red and plays GTA death sound).. It scared the shit out of me after that i literally had a panic attack... not sure if its just a haloween thing, but for the forseeable future laser is unplayable without nofail

**Comment by bdach** [comment]
You will be able to disable the red flash next release.

https://github.com/ppy/osu/discussions/15226#discussioncomment-1515838

**Comment by peppy** [comment]
@RadekMojzis sorry about that. do you have a medical condition which can trigger such reactions? also, you mention both the sound and the flash - was it both of these combined? if so we may need to consider making changes to ensure it doesn't affect other people in the same way.

as mentioned, the flash is toggleable in the update that just dropped, so please give that a try at lower volumes and see if it helps.

**Comment by RadekMojzis** [comment]
No, i do not have any medical conditions that would cause that (that im aware of), i do have trauma from childhood but nothing like this ever happens to me.
maybe its the combination of having the screen turning red as your hp gets low (i dont know why hp is a thing) and then being alone in the dark, having effects sounds much louder than music....
i was playing on normal client, then switched to laser because i wanted to practice with diff adjust... and then that happend
so maybe it was the combination of all those things


 If i had history of souch things happening to me i wouldnt have bothered reporting this, but this was basically the first time something terrifyed me this badly
 
 
_____________________________________________________________
> Od: "Dean Herbert" ***@***.***>
> Komu: "ppy/osu" ***@***.***>
> Datum: 27.10.2021 23:07
> Předmět: Re: [ppy/osu] New fail animation is dangerous (Discussion #15322)
>
 
@RadekMojzis <https://github.com/RadekMojzis> sorry about that. do you have a medical condition which can trigger such reactions? also, you mention both the sound and the flash - was it both of these combined? if so we may need to consider making changes to ensure it doesn't affect other people in the same way.
as mentioned, the flash is toggleable in the update that just dropped, so please give that a try at lower volumes and see if it helps.
—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub <https://github.com/ppy/osu/discussions/15322#discussioncomment-1548720>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AFKWTULR5WGUKQT4QFP6TPLUJBSZXANCNFSM5G22HP7A>.
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Comment by smoogipoo** [comment]
The flash intensity has been reduced a bit in https://github.com/ppy/osu/pull/15382. Audio levels are still yet to be addressed.
